core/trino-main/src/main/java/io/trino/sql/planner/plan/TableScanNode.java

```diff
@@ -51,12 +52,18 @@
 
     @Nullable // null on workers
     private final TupleDomain<ColumnHandle> enforcedConstraint;
+    @Nullable // null on workers
+    private final Optional<PlanNodeStatsEstimate> statistics;
```

Comment:
maybe call it `bestKnownStatistics` to show direct relation of the field and configuration variable.

Reply:
as discussed, i will change the config variable instead - `statistics-precalculation-for-pushdown.enabled`